samples/certs/Makefile

```diff
@@ -1,18 +1,26 @@
 .SUFFIXES: .csr .pem .conf
 .PRECIOUS: %/ca-key.pem %/ca-cert.pem %/cert-chain.pem
-#.PRECIOUS: %/cluster-ca.csr %/intermediate.conf
+.PRECIOUS: root-cert.csr root-ca.conf %/cluster-ca.csr %/intermediate.conf
```

Comment:
You have '.PRECIOUS` above. Isn't it overriding it?

Reply:
No, they are additive. Same as you can have multiple `.PHONY` lines